source/manual/manage-assets.html.md.erb

```diff
@@ -128,3 +128,28 @@ changing the URL, follow these steps:
    attachment_data = asset.assetable
    attachment_data.update(file_size: 123, number_of_pages: 28)
    ```
+
+## Getting an asset's ID
```

Comment:
If I'm being fussy could I ask you to link to the `## Getting an asset's` ID section in `## Redirecting an asset`, not a necessity but that's then I need it (unless it's not covered by the page fold, then it's all good :) )